lib/sync-DataSetModel.js

```diff
@@ -564,10 +629,15 @@ var self = {
 };
 
 var init = function (apiMetrics) {
+
```

Comment:
could we call defaultDataHandler.setAPIMetrics here instead of changing the signature?